scripts/bluepill.sh

```diff
@@ -82,12 +81,28 @@ bluepill_build()
           echo No bp built
           exit 1
   }
+  xcodebuild \
+    -workspace Bluepill.xcworkspace \
+    -scheme bp_test_dumper \
+    -configuration Release \
+    -sdk iphonesimulator12.2 \
+    -derivedDataPath "build/" | tee result.txt | $XCPRETTY
+  test $? == 0 || {
+          echo Build failed
+          xcodebuild -list -workspace Bluepill.xcworkspace
+          cat result.txt
+          exit 1
+  }
+  test -x build/Build/Products/Release/bluepill || {
+          echo No test dumper dynamic library built
```

Comment:
You're testing for `bluepill` but the error says no test dumper library built... one of the two is wrong.

Reply:
That's a typo. I will fix it.